### ISSUE TITLE
[bug] Original string offsets is not right for text_b for Chinese language sentence pair tokenization!

### ISSUE THREAD

**Issue by jia-zhuang** [huggingface/tokenizers#143]
In the issue [Offsets is not right for Chinese language sentences tokenization!](https://github.com/huggingface/tokenizers/issues/81), @n1t0 has added the function to map the token offset to the original string. It works for single sentence encode, but when I do sentence pair encode, the pair sentence(i.e. text_b) seems has misaligned original offsets.

```python
from tokenizers import BertWordPieceTokenizer
tokenizer = BertWordPieceTokenizer('./bert-base-chinese/vocab.txt')
text_a = "薛定谔（Schrödinger），奥地利著名物理学家。"
text_b = "他建立了波动力学。"
tokened = tokenizer.encode(text_a, text_b)

original_str = text_a + text_b
assert original_str == str(tokened.original_str)

for token, offset in zip(tokened.tokens, tokened.offsets):
    original_str_offset = tokened.original_str.offsets(offset)
    if original_str_offset is None:
        original_str_offset = (0, 0)
    print(token, '\t', original_str_offset, '\t', original_str[original_str_offset[0]: original_str_offset[1]])

# output
[CLS] 	 (0, 0) 	 
薛 	 (0, 1) 	 薛
定 	 (1, 2) 	 定
谔 	 (2, 3) 	 谔
（ 	 (3, 4) 	 （
sc 	 (4, 6) 	 Sc
##hr 	 (6, 8) 	 hr
##od 	 (8, 10) 	 öd
##ing 	 (10, 13) 	 ing
##er 	 (13, 15) 	 er
） 	 (15, 16) 	 ）
， 	 (16, 17) 	 ，
奥 	 (17, 18) 	 奥
地 	 (18, 19) 	 地
利 	 (19, 20) 	 利
著 	 (20, 21) 	 著
名 	 (21, 22) 	 名
物 	 (22, 23) 	 物
理 	 (23, 24) 	 理
学 	 (24, 25) 	 学
家 	 (25, 26) 	 家
。 	 (26, 27) 	 。
[SEP] 	 (0, 0) 	 
他 	 (51, 52) 	 
建 	 (52, 53) 	 
立 	 (53, 54) 	 
了 	 (54, 55) 	 
波 	 (55, 56) 	 
动 	 (56, 57) 	 
力 	 (57, 58) 	 
学 	 (58, 59) 	 
。 	 (59, 60) 	 
[SEP] 	 (0, 0) 
```

**Comment by n1t0** [comment]
This is now fixed (starting from any `0.7.0rc` version of the library). The offsets are now relative to the original string by default and starting back at 0 again for the pair sequence.

**Comment by jia-zhuang** [comment]
@n1t0 I've just test it, it's very convenient, thanks very much!
